src/aliceVision/matchingImageCollection/geometricFilterUtils.hpp

```diff
@@ -48,9 +48,8 @@ void fillMatricesWithUndistortFeaturesMatches(const matching::IndMatches &putati
 {
   typedef typename MatT::Scalar Scalar; // Output matrix type
 
-  const bool I_hasValidIntrinsics = cam_I && cam_I->isValid();
-  const bool J_hasValidIntrinsics = cam_J && cam_J->isValid();
-
+  const bool I_hasValidIntrinsics = cam_I && cam_I->isValid() && cam_I->hasDistortion();
```

Comment:
I agree that the wording is a bit complex but on the other hand, it is important to explain in the name that we are creating matrices instead of the standard data structures.

I renamed the other one: `MatchesPairToMat` which was ambiguous.